EventLoopScheduler test fails on Travis CI

```
================================== FAILURES ===================================
_____ TestEventLoopScheduler.test_event_loop_schedule_ordered_actions_due _____
self = <tests.test_concurrency.test_eventloopscheduler.TestEventLoopScheduler testMethod=test_event_loop_schedule_ordered_actions_due>
    def test_event_loop_schedule_ordered_actions_due(self):
        scheduler = EventLoopScheduler(exit_if_empty=True)
        gate = threading.Semaphore(0)
        result = []
    
        def action(scheduler, state):
            result.append(3)
            gate.release()
    
        scheduler.schedule_relative(0.10, action)
        scheduler.schedule_relative(0.05, lambda s, t: result.append(2))
        scheduler.schedule(lambda s, t: result.append(1))
    
        gate.acquire()
>       assert result == [1, 2, 3]
E       AssertionError: assert [1, 3] == [1, 2, 3]
E         At index 1 diff: 3 != 2
E         Right contains more items, first extra item: 3
E         Use -v to get the full diff
tests\test_concurrency\test_eventloopscheduler.py:77: AssertionError
```

https://travis-ci.org/ReactiveX/RxPY/jobs/513968253

--- a/tests/test_concurrency/test_eventloopscheduler.py
+++ b/tests/test_concurrency/test_eventloopscheduler.py
@@ -69,8 +69,8 @@ class TestEventLoopScheduler(unittest.TestCase):
             result.append(3)
             gate.release()
 
-        scheduler.schedule_relative(0.10, action)
-        scheduler.schedule_relative(0.05, lambda s, t: result.append(2))
+        scheduler.schedule_relative(0.2, action)
+        scheduler.schedule_relative(0.1, lambda s, t: result.append(2))
         scheduler.schedule(lambda s, t: result.append(1))
 
         gate.acquire()

